### PR TITLE
Update office-365-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
The options have changed, added steps reflecting the new options to access groups activity

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2932

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/admin/activity-reports/office-365-groups.md [hunk 1]
```diff
@@ -32,8 +32,7 @@ The Microsoft 365 **Reports** dashboard shows you the activity overview across t
 
 1. In the admin center, go to the **Reports** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2074756" target="_blank">Usage</a> page.
 
-    
-2. From the **Select a report** drop-down, select **Office 365** \> **Groups activity**.
+2. From the options, Select **View more** under **Active users - Microsoft 365 Services****, then Select a report** drop-down, select **Office 365** \> **Groups activity**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/office-365-groups.md]:
```suggestion
2. From the options, select **View more** under **Active users - Microsoft 365 Services**.
3. From the **Select a report drop-down**, select **Office 365** \> **Groups activity**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/admin/activity-reports/office-365-groups.md [hunk 2]
```diff
@@ -32,7 +32,8 @@ The Microsoft 365 **Reports** dashboard shows you the activity overview across t
 
 1. In the admin center, go to the **Reports** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2074756" target="_blank">Usage</a> page.
 
-2. From the options, Select **View more** under **Active users - Microsoft 365 Services****, then Select a report** drop-down, select **Office 365** \> **Groups activity**.
+2. From the options, select **View more** under **Active users - Microsoft 365 Services**.
+3. From the **Select a report drop-down**, select **Office 365** \> **Groups activity**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/office-365-groups.md]:
```suggestion
3. From the **Select a report** drop-down, select **Office 365** \> **Groups activity**.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - one more, please.

**Pull request merged**